### PR TITLE
PFM-ISSUE-5805 added parameter --chromeDriverVersion

### PR DESCRIPTION
**Pull request by stefancplace**
Adds a parameter --chromeDriverVersion and sets Chrome Driver Version default to 87.0.4280.20

[PFM-ISSUE-5805](https://base.cplace.io/pages/tzi617fhz5gjr9lzp07awvry7/PFM-ISSUE-5805-WDIO-not-compatible-with-latest-Chrome#id_wsrqdlqi9cus7fztqfsfa3gnz=cf.cplace.cfactoryPlatform.overview)

### CODE REVIEW

File: src/commands/e2e/E2E.ts [hunk 1]
```diff
@@ -18,6 +18,7 @@ export class E2E implements ICommand {
     public static readonly ALLURE_PACKAGE_NAME: string = '@wdio/allure-reporter';
     public static readonly IMAGE_COMPARISON_PACKAGE_NAME: string = 'wdio-image-comparison-service';
     public static readonly DEV_TOOLS_PACKAGE_NAME: string = '@wdio/devtools-service';
+    public static readonly DEFAULT_CHROMEDRIVERVERSION: string = '87.0.4280.20';
```
[the remark this answers is not in the document]
Comment (stefancplace) [reply src/commands/e2e/E2E.ts:21]: what do you mean with current behaviour? Would you suggest setting default version to 85.0.4183.87?
[Here](https://chromedriver.storage.googleapis.com/) you can find the list of available chrome drivers

[the remark this answers is not in the document]
Comment (stefancplace) [reply src/commands/e2e/E2E.ts:21]: Current behaviour is that default ChromeDriverVersion is set by package selenium-standalone via their default configuration.
See here https://github.com/vvo/selenium-standalone/pull/486/files

[the remark this answers is not in the document]
Comment (stefancplace) [reply src/commands/e2e/E2E.ts:21]: So right I guess the root problem is that the underlying selenium-standalone package does not support detecting latest Chrome Driver. They actually have a [Feature Request](https://github.com/vvo/selenium-standalone/issues/490) for that.

[the remark this answers is not in the document]
Comment (stophi-dev) [reply src/commands/e2e/E2E.ts:21]: In most cases it makes sense to use the latest Chrome version that is available on the system. I thought that would be the default behavior.
I don't like the idea that we always have to update cplace-cli whenever a new Chrome version is released. We might forget about it and finally run the tests with a totally outdated Chrome driver.

[the remark this answers is not in the document]
Comment (stefancplace) [reply src/commands/e2e/E2E.ts:21]: I totally agree unfortunately we do not really have that in our hands. Would you suggest tackling the issue in selenium-standalone package by implementing a scanner mechanism that automatically downloads the latest Chromedriver ?

[the remark this answers is not in the document]
Comment (stefancplace) [reply src/commands/e2e/E2E.ts:21]: [selenium-standalone](https://github.com/vvo/selenium-standalone/releases/tag/v6.22.0) now supports automatic detection of latest version. This should be the default behaviour also in cplace-cli and would like to keep the possibility to adjust driver version via --chromeDriverVersion

**Comment by stefancplace** [comment]
WDIO is capable of utilizing latest Chrome version by now. In case we have problems once again we can reopen this PR.

**Pull request closed without merging**